static/oas/v1/openapi.yml

```diff
@@ -20,6 +20,10 @@ paths:
     $ref: "./stats/penalties.yml#/paths/penalties"
   /user:
     $ref: "./users/users.yml#/paths/user"
+  /teams:
+    $ref: "./team/team.yml#/paths/teams"
+  # /teams/{id}:
+  #   $ref: "./team/team.yml#/paths/~1team~1%7Bid%7D"
```

Comment:
Can we just remove these two lines of comments? If further work will need to be completed in the future we can create an issue instead of leaving comments in the code.

Reply:
These were removed and fixed to work with the program.